packages/definitions-parser/test/packages.test.ts

```diff
@@ -180,6 +180,30 @@ describe(TypingsData, () => {
     expect(data.isNotNeeded()).toBe(false);
   });
 
+  describe("project", () => {
+    it("returns the name of the only project", () => {
+      expect(data.projectName).toBe("zombo.com");
+    });
+
+    it("returns the versioned name if not latest", () => {
```

Comment:
Should this return the version numbers? This line suggests it does, but the expected output is just the names of the projects.